app/views/forms/components/repayment_input_checkbox.scala.html

```diff
@@ -28,23 +28,29 @@
 @field = @{formData("repaymentClaim")}
 
 <div class="govuk-form-group">
-    <fieldset class="govuk-fieldset" aria-describedby="expiry-date-hint">
+    <fieldset class="govuk-fieldset" aria-describedby="@{field.id}">
```

Comment:
same bug here with checkbox it doesnt remember that it is checked or it is not checking it with there is data inside.

I cant find exact reason here but on http://localhost:9581/manage-tariff-classifications/cases/v2/800000013/liability/edit-details and you have some data in dvr number which is under repayment claim checkbox it doesnt not check it to expand the input to be able to delete even the data is inside, it doesnt not remember or there is not aria-expanded attribute on tag,
there should be aria-expanded="true" but it always false even when there is data inside or there is error

I dont see in old code this in place explicitly but it is working fine there, feel free to play around maybe something else is missing which I could not figure out

Reply:
reverting to old version not exactly sure either where it is failing, thanks for spotting this.

Reply:
hopefully should remember the checked status now reverted back/similar to old code